packages/dataviews/src/add-filter.js

```diff
@@ -107,50 +110,45 @@ export default function AddFilter( { filters, view, onChangeView } ) {
 							>
 								<WithSeparators>
 									<DropdownMenuGroup>
-										{ filter.elements.map( ( element ) => (
-											<DropdownMenuItem
-												key={ element.value }
-												role="menuitemradio"
-												aria-checked={
-													activeElement?.value ===
-													element.value
-												}
-												prefix={
-													activeElement?.value ===
-														element.value && (
-														<Icon icon={ check } />
-													)
-												}
-												onSelect={ ( event ) => {
-													event.preventDefault();
-													onChangeView(
-														( currentView ) => ( {
-															...currentView,
+										{ filter.elements.map( ( element ) => {
+											const isActive =
+												activeElement?.value ===
+												element.value;
+											return (
+												<DropdownMenuItem
```

Comment:
Apologies if this was already discussed, but is there a reason why we're not using `DropdownMenuItemRadio`? It should provide the right semantics out of the box, including showing/hiding the icon

Reply:
The issue was that radio items cannot be unset after they've been set, which is not the behavior we want here (user should be able to unset the value).

Reply:
Got it, thank you for the reminder